docs/ops.html

```diff
@@ -3946,6 +3946,7 @@ <h3>Enter Migration Mode on the Brokers</h3>
   </p>
 
   <ul>
+    <li><a href="#brokerconfigs_broker.id">broker.id</a>: Ensure <code>broker.id</code> is set to a non-negative integer even if <code>broker.id.generation.enable</code> is enabled (default is enabled).</li>
```

Comment:
That is another issue I mentioned that the broker id is in the reserved range if it was generated. As we encourage users to set broker.id, maybe we should describe the possible error to save users' life 😄

Reply:
Sounds good. Thanks for pointing it out.

Reply:
I will do that 🦸🏼